pull/README.md

```diff
@@ -0,0 +1,282 @@
+# Remote Provider Pull
+
+The `pull` addon was recently a core command but has been pulled out due as it is sometimes 
+quicker to get updates to just the addon as opposed to the whole platform.
```

Comment:
The `pull` addon was recently a core command but has been moved to an addon command.